src/dirs.rs

```diff
@@ -42,6 +31,27 @@ impl BatProjectDirs {
         })
     }
 
+    pub fn get_cache_dir() -> Option<PathBuf> {
+        // on all OS prefer BAT_CACHE_PATH if set
+        let cache_dir_op = env::var_os("BAT_CACHE_PATH")
+            .map(PathBuf::from)
+            .filter(|p| p.is_absolute());
+        if !cache_dir_op.is_none() {
```

Comment:
you can use:
```rust
if cache_dir_op.is_some() {
  // …
```

Reply:
Nice! If it wasn't obvious already, I'm still learning Rust.